### PR TITLE
feat(sns): Add indication of whether a canister is frozen in `sns health`

### PR DESCRIPTION
**Pull request by anchpop**
Example output:

| Name                    | Memory              | Cycles                                                                                                                         | Upgrades Remaining |
|-------------------------|---------------------|--------------------------------------------------------------------------------------------------------------------------------|--------------------|
| BOOM DAO                | 👍                   | ❌ governance: (6.96 TC), index: (0.00 TC 🥶) | 24                 |
| CYCLES-TRANSFER-STATION | 👍                   | 👍                                                                             | 8                  |
| Catalyze                | 👍                   | ❌ governance: (9.76 TC), swap: (8.83 TC), archive: (9.30 TC)                  | 1                  |

(This is using test data, BOOM DAO's index canister is fine)

The benefit of this is that it draws attention to the canisters that are really in trouble - the frozen ones.

The definition of frozen used by the tool is: the number of cycles is below the freezing threshold (as reported by the canister summary)

### CODE REVIEW

File: rs/sns/cli/src/health.rs [hunk 1]
```diff
@@ -57,11 +68,16 @@ impl TableRow for SnsHealthInfo {
...
             .iter()
-            .filter(|(cycles, _)| (*cycles as f64) < 10.0 * 1000.0 * 1000.0 * 1000.0 * 1000.0)
+            .filter(|(cycles, _)| (cycles.cycles as f64) < CYCLES_THRESHOLD_TC * TC)
             .map(|(cycles, canister_type)| {
                 format!(
-                    "{canister_type}: ({:.2} TC)",
-                    *cycles as f64 / 1000.0 / 1000.0 / 1000.0 / 1000.0
+                    "{canister_type}: ({:.2} TC{frozen})",
+                    cycles.cycles as f64 / TC,
+                    frozen = if (cycles.freezing_threshold as u128) > cycles.cycles {
```
Comment (aterga) [inline rs/sns/cli/src/health.rs:76]: Nit. Such a strange way to write this condition. 

```suggestion
                    frozen = if cycles.cycles < cycles.freezing_threshold as u128 {
```

**Comment by aterga** [comment]
> ❌ governance: (9.76 TC), swap: (8.83 TC), archive: (9.30 TC)

Why does the red cross appear in this example if the canisters aren't frozen?

File: rs/sns/cli/src/health.rs (hunk 1 above)
Comment (anchpop) [reply rs/sns/cli/src/health.rs]: Done!

**Comment by anchpop** [comment]
> > ❌ governance: (9.76 TC), swap: (8.83 TC), archive: (9.30 TC)
> 
> Why does the red cross appear in this example if the canisters aren't frozen?

My intention when writing this tool is that a red cross would appear if an SNS was not obviously-healthy. So a red cross appears if a canister has less than 10T cycles, since 10T is really not very many cycles. ([1 ICP is about 9TC](https://dashboard.internetcomputer.org/), so expecting SNS communities to keep their canisters above 10TC seems reasonable).

**Review by max-dfinity — APPROVED** [review]

**Pull request merged**